src/App.js

```diff
@@ -7,27 +7,28 @@ import {
   Route,
 } from 'react-router-dom';
 import apolloClient from './apollo';
-import English from './English';
-import Portuguese from './Portuguese';
-import Spanish from './Spanish';
-import Home from './Home';
+import Home from './Home/Home';
+import Login from './Login/Login';
+import Register from './Register/Register';
+import NotFound from './NotFound/NotFound';
```

Comment:
I would suggest renaming all the page files above to `../index.js` which will shorten the import path to the folder path, it makes sense like that since each feature folder should have a single entry point which in this case would be the page file of the feature.

```
import Home from './Home';
import Login from './Login';
import Register from './Register';
import NotFound from './NotFound';
```